docs/customizations/icon-cache.md

```diff
@@ -0,0 +1,29 @@
+# Icon Cache
+
+Homarr automatically loads icons for many services, though directly links to an external server. To reduce network requests and use homarr offline, a local cache can be used.
+
+Homarr has no automatic icon caching. The script below can be used to update an icon cache.
+
+## Icon Cache Script
+
+On a system with access to the homarr directory, run the following script. First update the variable with the path to homarr's docker directory.
```

Comment:
```suggestion
To run the script, you need to open the terminal which has access to Homarr's directory. Usually, this is on your host machine or inside your docker container / volume.
Replace ``/Volumes/docker/homarr`` with your directory of Homarr.

:::

warn

Make a backup, before you execute any scripts, in case anything breaks!

::: 
```

Reply:
The script does create a backup, but suppose this is good practice anytime you're running a script anyway.